docs/development/contributor_guidelines.rst

```diff
@@ -66,7 +66,7 @@ our HTML templates in ``securedrop/source_templates`` and
 
   .. code:: sh
 
-      html_lint.py --printfilename --disable=optional_tag,extra_whitespace,indentation example.html
+      make html-lint
```

Comment:
Let's add a similar note to the flake8 section (that you can run `make flake8` to check all the covered code at once)